### DISCUSSION TITLE
如何解决cookie共享问题

### DISCUSSION THREAD

**Discussion by llbiao** [jd-opensource/micro-app#91] [General]
场景：
每个子应用有独立的登录，登录完会将token写入到cookie里面，主应用如何共享子应用的cookie？

**Comment by bailicangdu** [comment]
本身就是共享的，cookie没有隔离

**Reply by llbiao** [reply]
这样怎么写入呢？
![image](https://user-images.githubusercontent.com/32304163/133251665-c4dc7144-c3e1-4bf8-9a78-03c81d4aa75c.png)

**Comment by bailicangdu** [comment]
通过接口下发的set-cookie写入的吗，正常执行就行
